config/routes.rb

```diff
@@ -164,9 +164,9 @@ def put(*_args); end
   end
 
   scope :as => :internal, :module => "internal", :path => "internal" do
-    routing_helper.redirect_major_version("v0.0", "internal", :via => [:get])
+    routing_helper.redirect_major_version("v1.0", "internal", :via => [:get])
```

Comment:
Should we also add a redirect for `/v0.0` to `/v1.0`?

Reply:
It's the same content that it used to be and I think it should be at v1.0.  In the future, I don't expect it to match the versioning of the public API.

Reply:
Actually the reply will be different because the v0.0 version didn't use pagination